pkg/clusterd/inventory/disk.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/rook/rook/pkg/util/sys"
 )
 
+var (
+	isRBD = regexp.MustCompile("^/dev/rbd[0-9]+p?[0-9]{0,}$")
```

Comment:
i believe the devices actually don't have the `/dev/` prefix at this point. After any further changes, it would be great if you could squash to a single commit.